src/components/dashboard/Stats.js

```diff
@@ -72,39 +71,45 @@ class Stats extends PureComponent {
       <div>
         <div className="row">
           <div className="col-md-4 col-sm-4 txt-c">
-            <div className="rka-box no-min-height">
-              <div className="txt-xl pb-sm txt-brand">{totals.traffic.toLocaleString()} {symbols.traffic}</div>
+            <div className="rka-box rka-box-dashboard-short no-min-height">
+              <div className="txt-xl pb-sm txt-brand">
+                {totals ? totals.traffic.toLocaleString() + symbols.traffic : '-'}
```

Comment:
there's a space missing most likely between the value and the symbol. you can use `${totals.traffic.toLocaleString()} ${symbols.traffic}`